docs/sql-ref-syntax-ddl-repair-table.md

```diff
@@ -19,4 +19,45 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+`MSCK REPAIR TABLE` recovers all the partitions in the directory of a table and updates the Hive metastore. When creating a table using `partitoned by` clause, partitions are generated and registered in the Hive metastore. However, if the partitioned table is created from existing data, partitions are not registered automatically in the Hive metastore. User needs to run MSCK REPAIR TABLE to register the partitions. `MSCK REPAIR TABLE` on a non-existent table or a table without partitions throws an exception. Another way to recover partitions is to use `ALTER TABLE RECOVER PARTITIONS`.
```

Comment:
(Pardon me if this is duplicated - thought I left this comment but doesn't look like it now.)
Maybe back-tick the last occurrence of "MSCK REPAIR TABLE"? 
and maybe `partitioned by` -> `PARTITIONED BY`